doc/usecases/draft/copy_paste_between_users.md

```diff
@@ -0,0 +1,38 @@
+# Use Case: Copy & Paste of Selection Between Maps of Different Users
```

Comment:
Do we really need this UC? I don't see how this differs in the use case description or in the implementation to two maps of the same user (except of permissions, which currently are not implemented anyway, #480)?